airbase/pom.xml

```diff
@@ -447,6 +445,8 @@
                             -XX:+HeapDumpOnOutOfMemoryError
                             -XX:-OmitStackTraceInFastThrow
                             ${air.test.jvm.additional-arguments}
+                            -Duser.language=${air.test.language}
+                            -Duser.region=${air.test.region}
```

Comment:
keep `air.test.jvm.additional-arguments` last

Reply:
We should put those above with `-Dfile.encoding=`

Reply:
Done!